go.mod

```diff
@@ -4,6 +4,11 @@ require (
 	github.com/fullstorydev/grpcurl v1.2.1
 	github.com/golang/protobuf v1.3.1 // indirect
 	github.com/jhump/protoreflect v1.2.0
+	github.com/spf13/cobra v0.0.5
+	github.com/stretchr/testify v1.4.0
+	golang.org/dl v0.0.0-20190926113319-b3a3479952ba // indirect
 	golang.org/x/net v0.0.0-20190424112056-4829fb13d2c6
 	google.golang.org/grpc v1.20.1
 )
+
+go 1.13
```

Comment:
Was this accidental? If not we're using go 1.12 atm, can we change this or update project to use 1.13?